lib/httpServer.js

```diff
@@ -31,15 +31,18 @@ module.exports = function (provider, logger) {
           break;
         case "POST":
           //console.log("Request coming in:", body);
+          
+          var send400 = function() {
```

Comment:
This functionality was sort of a last resort measure for unhandled errors. Sadly it breaks JSON-RPC 2.0, as all Requests MUST return a Response, which doesn't happen in this case.